packages/@vuepress/theme-default/src/client/components/ToggleDarkButton.vue

```diff
@@ -49,12 +49,27 @@ const setDarkClass = (): void => {
   htmlEl?.classList.toggle('dark', isDark.value)
 }
 
+const updateThemeColor = (): void => {
+  const themeColorEl = window?.document.querySelector(
+    'meta[name="theme-color"]'
+  )
```

Comment:
If we are going to support mulptiple `meta[name="theme-color"]`, the code here should be updated, too.

Reply:
https://github.com/vuepress/vuepress-next/blob/b0a9815c9eeaa2ba623cc1a153139e85397d450e/packages/%40vuepress/theme-default/src/client/components/ToggleDarkButton.vue#L52-L62

Now is able to change color for all duplicated 'theme-color' meta tags. I just simply change both light and dark specialized meta tags, because if only change the 'theme-color' meta tag based on current system side color mode would let it not working in some situation:

- system side color mode is currently **light**, if only change the 'theme-color' meta tag matched with`(prefers-color-scheme: light)` to **dark**, then change system side color mode to **dark**, 'theme-color' meta tag matched with`(prefers-color-scheme: light)` is still **dark**, then switch auto enabled dark mode  to light mode by the website's button and change system side color mode back to **light**, the `isDark.value` is updated but not changed, so 'theme-color' meta tag matched with`(prefers-color-scheme: light)` is still in **dark**.

To solve this would be little complicated, personally I think change them both would be better.